src/test/java/fi/helsinki/cs/tmc/core/commands/DownloadExercisesTest.java

```diff
@@ -53,6 +53,9 @@
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.After;
```

Comment:
Incorrect order of imports,  the import order and grouping should be:
- All static imports in a single group
- fi.helsinki.cs.tmc imports (only if this source file is in the fi.helsinki.cs.tmc package space)
- Third-party imports, one group per top-level package, in ASCII sort order
- for example: android, com, junit, org, sun
- java imports
- javax imports